content/js/asset.md

```diff
@@ -1,55 +1,32 @@
-## Uploading an asset by base64 data
-
-```js
-// node.js
-fs.readFile('my-awesome-picture.jpg', (err, data) => {
-  if (err) {
-    throw err;
-  }
-
-  let profilePic = new skygear.Asset({data: data});
-  // Profile = skygear.Record.extend('profile');
-  let profile = new Profile({
-    name: 'Handsome Guy',
-    picture: profilePic
-  });
-  skygear.publicDB.save(profile).then((p) => {
-    // fetched record field will have a `url` attribute available
-    console.log('Download my handsome selfie here => %s', p.picture.url);
-  });
-})
-```
-
-## Creating an asset from `input` element
+For file upload to work properly, please set `ASSET_STORE_URL_PREFIX` environment
+variable to have value `https://<your-app-name>.skygeario.com/files` (which is
+simply your Server EndPoint plus `files` at the end) on Skygear
+[Portal](https://portal.skygear.io/app/info) Settings page. For more environment variable
```

Comment:
the hyperlink should be added for the whole phrase "Skygear Portal Settings page"